meta/sai_serialize.h

```diff
@@ -138,6 +138,9 @@ std::string sai_serialize_ipmc_entry_type(
 std::string sai_serialize_qos_map_item(
         _In_ const sai_qos_map_t& qosmap);
 
+std::string sai_serialize_switch_attr(
```

Comment:
please remove we already have generic function to serialize attributes for all objects